test/conftest.py

```diff
@@ -14,6 +19,10 @@
     CodeSegment,
 )
 from sqlfluff.core.templaters import TemplatedFile
+from .dialects.parse_fixtures import load_file
```

Comment:
Not sure why this needs to be relative to current path (`.`), rather than relative to this file location, which seems to be causing you your issue:

```suggestion
from dialects.parse_fixtures import load_file
```

Reply:
Actually ignore this. This causes other errors.